lib/src/fields/form_builder_checkbox_group.dart

```diff
@@ -25,7 +26,7 @@ class FormBuilderCheckboxGroup<T> extends FormBuilderFieldDecoration<List<T>> {
   final OptionsOrientation orientation;
 
   /// Creates a list of Checkboxes for selecting multiple options
-  FormBuilderCheckboxGroup({
+  FormBuilderCheckboxGroup(this.visualDensity, {
```

Comment:
Move inside {} to convert this to parameter named

Reply:
My bad! Done Please check now.

Reply:
My Bad! Done. Please check.

Reply:
Did you push this changes to version 9.1.1? @deandreamatias